README.md

```diff
@@ -16,19 +20,20 @@ Each service might have a number of libraries available from each of the followi
 
 ### Client: New Releases
 
-New wave of packages that we are announcing as **GA** and several that are currently releasing in **preview** on `npm`. These libraries allow you to use and consume existing resources and interact with them. These libraries share a number of core functionalities found in the Azure Core package such as retries, logging, transport protocols, authentication protocols, etc. Learn more about these libraries by reading [the guidelines](https://azure.github.io/azure-sdk/typescript/guidelines/) that they follow.
+Given an Azure resource already exists, you would use the client libraries to consume it and interact with it.
+The new wave of packages that we are actively working on follow the [Azure SDK Design Guidelines for JavaScript & TypeScript](https://azure.github.io/azure-sdk/typescript_introduction.html). These packages share a number of core functionalities such as retries, logging, transport protocols, authentication protocols, etc.
```

Comment:
As said before, I would look forward and consider track2 as normal and not new anymore.

Reply:
Updated in https://github.com/Azure/azure-sdk-for-js/pull/13255/commits/4da82a8ee5e14172d3a42989b900ba494c821591 to no longer differentiate between old and new libraries